Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -67,12 +85,12 @@ public static LootLockerConfig current
         public string domainKey;
         [HideInInspector]
         public int gameID;
-        public string game_version = "1.0";
+        public string game_version = "1.0.0.0";
         [HideInInspector]
         public string deviceID = "defaultPlayerId";
         public platformType platform;
         public enum platformType { Android, iOS, Steam, PlayStationNetwork }
-        public bool developmentMode;
+        public bool developmentMode = true;
```

Comment:
This is a semi-deprecated property due to Multiple API Keys, so I don't think we should have it default to true.

Reply:
If this is set to false, using a "new" API-key will fail since it is expecting a live-key. We still have stage/live environments and development-mode on/off, you can just create multiple ones. As I understand it, we want to remove stage/live later on, but as it is now, development mode on/off is still needed. And same as above, this was defaulted to developmentMode = true in the previous ConfigAsset.